Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.19 as build
+
+ENV CGO_ENABLED=0
```

Comment:
I saw this in the [.goreleaser](https://github.com/exercism/cli/blob/8e73ec3131e8218c0ff31d5e0d1a050cabd6864d/.goreleaser.yml#L6) file. Not sure if this is really necessary.